app/lib/formatter.rb

```diff
@@ -95,6 +95,6 @@ def hashtag_html(match)
   end
 
   def mention_html(match, account)
-    "#{match.split('@').first}<span class=\"h-card\"><a href=\"#{TagManager.instance.url_for(account)}\" class=\"u-url mention\">@#{account.username}</a></span>"
+    "#{match.split('@').first}<span class=\"h-card\"><a href=\"#{TagManager.instance.url_for(account)}\" class=\"u-url mention\">@<span>#{account.username}</span></a></span>"
```

Comment:
Well, the h-card stuff has been added by microformat folks, but the span inside the a has a visual role (underline it on hover but do not underline the @ sign)